networkx/algorithms/approximation/broadcasting.py

```diff
@@ -0,0 +1,139 @@
+import networkx as nx
+from networkx import NetworkXError
+from networkx.utils import not_implemented_for
+
+__all__ = [
+    "tree_broadcast_center",
+    "tree_broadcast_time",
+]
+
+
+def _get_max_broadcast_value(G, U, v, values):
+    adj = sorted(set(G.neighbors(v)) & U, key=lambda u: values[u], reverse=True)
```

Comment:
Can you check if it speeds things up to use `key=values.get` instead of the `lambda` function?  If so, it should help in the next function too.

Reply:
I tested on some larger test cases, and there was a slight performance boost. (From ``5.9s`` to ``5.6s``)
Also it looks much cleaner, so I think we should go forward with the changes.